frontend/src/components/Scopes.tsx

```diff
@@ -52,7 +52,7 @@ const Scopes: React.FC<ScopesProps> = ({
   onScopesChanged,
 }) => {
   const fullAccessScopes: Scope[] = React.useMemo(() => {
-    return [...capabilities.scopes];
+    return capabilities.scopes.filter((scope) => scope !== "superuser");
```

Comment:
(backend) The capabilities will never have this, we should return `"superuser"` in `api.GetWalletCapabilities`

(or add `create_connection` in `lnclient.GetSupportedNIP47Methods` however I don't have a good feeling about this as it is rather a _Hub_ supported nip47 method)

Reply:
I have undone this change, thanks